webapp/client/src/components/StepForm.js

```diff
@@ -0,0 +1,34 @@
+import PropTypes from "prop-types";
+import FormNavButtons from "./FormNavButtons";
+
+export default function StepForm({
+  submitUrl,
+  csrfToken,
+  explanatoryButtonText,
+  overviewButton,
+  nextButtonLabel,
+}) {
+  return (
+    <form
+      noValidate
+      className="container px-0 form-container"
+      method="POST"
+      action={submitUrl}
+    >
+      <input type="hidden" name="csrf_token" value={csrfToken} />
+      <FormNavButtons
+        explanatoryButtonText={explanatoryButtonText}
+        overviewButton={overviewButton}
+        nextButtonLabel={nextButtonLabel}
+      />
+    </form>
+  );
+}
```

Comment:
How would we get additional fields into the form? As far as I see it, there is no argument that goes between the csrf and the nav buttons?

Reply:
This shall be revealed in the next PR :) (Hint: the answer is [children](https://reactjs.org/docs/jsx-in-depth.html#children-in-jsx))

Reply:
Hahaha cliffhanger alert! :D